components/admin_console/system_user_detail/team_list/team_row.tsx

```diff
@@ -11,14 +10,16 @@ import TeamIcon from 'components/widgets/team_icon/team_icon';
 import './team_row.scss';
 import TeamListDropdown from './team_list_dropdown';
 
-export default class TeamRow extends React.Component {
-    static propTypes = {
-        team: PropTypes.object.isRequired,
-        doRemoveUserFromTeam: PropTypes.func.isRequired,
-        doMakeUserTeamAdmin: PropTypes.func.isRequired,
-        doMakeUserTeamMember: PropTypes.func.isRequired,
-    };
-    renderTeamType = (team) => {
+type Props = {
+    team: {[x: string]: string};
+    onRowClick?: () => void;
+    doRemoveUserFromTeam: (teamId: string) => Promise<void>;
+    doMakeUserTeamAdmin: (teamId: string) => Promise<void>;
+    doMakeUserTeamMember: (teamId: string) => Promise<void>;
+}
+
+export default class TeamRow extends React.Component<Props, {}> {
+    private renderTeamType = (team: {[x: string]: string}): JSX.Element => {
```

Comment:
@devinbinnie sorry to bother you, but i just came upon something. Removing the return type from this function doesn't cause the deleting problem from running `make i18n-extract`

Reply:
That's odd, not sure why that is. Is it working now with the updated `mmjstool` regardless of the return type?

Reply:
No it isn't.

Reply:
Okay go ahead and remove the return type then. It shouldn't be needed anyway since it can be inferred.
We'll have to look into what `i18n-extract` is doing with respect to typescript files.
Thanks for investigating :)

Reply:
I'm done with it